src/assets/style/landmark/_stripView.less

```diff
@@ -26,15 +26,15 @@
     top: @spacing-2x;
     bottom: @spacing-2x;
     left: @spacing-2x;
+    display: flex;
+    flex-direction: column-reverse;
     overflow-y: scroll;
```

Comment:
Speaking of this, the scroll direction seems to be reversed in the strip bay. Any idea why that would be, and if we are able to quickly swap it around? It hardly matters, but a few times I've found myself going the wrong way because it's a nonstandard scroll.

Reply:
yes, it is _'reversed'_ 😉 

In fact, the way it was before was the real reversed version.  In order to achieve the first-item-displays-at-the-bottom, we had to get creative.  First we rotated the list 180 degress which means every strip is upside down.  Then we'd rotate every strip -180 degrees so then every strip was right side up again.

we don't need to do any of that craziness now that `flexbox` is much more supported.  The two changes above are all we need to achieve the same effect.

Reply:
Stupid alert: I'm just now seeing `flex-direction: column-reverse` literally one line above the one I commented on. I guess that's the one we want. If we can get that thing to scroll the standard way, just like a regular webpage, that would be peachy swell 😄

Reply:
That's a little bit harder to do.  Unfortunately, not in the scope of this PR.  Before We refactored the strip view, I believe it did scroll in the same direction it is now.

Reply:
👍 No need to go after field mice anyway so disregard